Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -1569,7 +1572,9 @@ private void writeToOutputStream(FacesContext facesContext, File file, String fi
         response.setHeader("Content-Disposition", "attachment;filename=\"" + fileName + "\"");
 
         ServletOutputStream outputStream = response.getOutputStream();
-        byte[] bytes = IOUtils.toByteArray(new FileInputStream(file));
+        FileInputStream fileInputStream = new FileInputStream(file);
+        byte[] bytes = IOUtils.toByteArray(fileInputStream);
+        fileInputStream.close();
         outputStream.write(bytes);
         outputStream.flush();
```

Comment:
`outputStream` would never be closed after and let a open handle back.

Reply:
Please ignore my comment. I did not see that `outputStream` is taken from `response` object and `outputStream` should not closed here.